fv3config/fv3run/_kubernetes.py

```diff
@@ -89,7 +88,6 @@ def _get_job(
     image_pull_policy="IfNotPresent",
     job_labels=None,
 ):
-    _ensure_locations_are_remote(config_location, outdir)
```

Comment:
Can you print some kind of warning if outdir is remote? Since presently it basically means "actually set no outdir". Alternatively allow outdir=None or remote, and if it's None use a specific local path in /var/tmp or something.

Reply:
added a warning.